actions/transactions/new.go

```diff
@@ -61,3 +61,19 @@ func (a *Action) newTransaction(c *gin.Context) {
 	contract := mappings.MapToDraftTransactionContract(transaction)
 	c.JSON(http.StatusCreated, contract)
 }
+
+// newTransaction will create a new transaction draft
+// New transaction draft godoc
+// @Summary		New transaction draft
+// @Description	New transaction draft
+// @Tags		New Transactions
```

Comment:
IMHO it should still be `Transactions`

Reply:
The idea was to have separate groups. Fixed.